examples/inventory-local-rs-testnet.yml

```diff
@@ -31,7 +31,7 @@ all:
         node_user: consumer
         # Modify the lines below up to the airdrop accounts if you want to use a different chain
         chain_repository: https://github.com/cosmos/interchain-security.git
-        chain_version: v1.0.0
+        chain_version: v1.2.1
```

Comment:
@ASoTNetworks did you test running the play with this version?

Reply:
Testing with that version it returns `panic: value from ParamSetPair is invalid: : decimal string cannot be empty`

Will revert back to v1.0.0